models/InviteDetail.v1.yaml

```diff
@@ -111,6 +111,13 @@ x-examples:
         watchlist_level: some text
         created_at: '2018-02-10T09:30Z'
         updated_at: '2018-02-10T09:30Z'
+      invites_parking_lot:
```

Comment:
I thought the property was `assigned_stall`. Did it change? 🤔

Reply:
this section is for the response.